packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -53,16 +53,22 @@ describe("Garbage Collection Tests", () => {
     // Update this for individual node to update the initial GC state of that node.
     const emptyGCDetails: IGarbageCollectionSummaryDetails = {};
 
+    // The flag for the close call
+    let closeCalled = false;
+
     const createGarbageCollector = (
         baseSnapshot: ISnapshotTree | undefined = undefined,
         getNodeGCDetails: (id: string) => IGarbageCollectionSummaryDetails = () => emptyGCDetails,
     ) => {
         mockLogger = new MockLogger();
         return GarbageCollector.create(
             gcRuntime,
-            { gcAllowed: true, deleteTimeoutMs },
+            { gcAllowed: true, deleteTimeoutMs, gcTestSessionTimeoutMs: 0 },
             (unusedRoutes: string[]) => {},
             () => Date.now(),
+            () => {
```

Comment:
nit: this can be in one line () => { closeCalled = true; }. Just makes it look a little cleaner.